libs/designer/src/lib/ui/templates/basics/resourcepicker.tsx

```diff
@@ -138,9 +139,9 @@ export const ResourcePicker = ({ viewMode = 'default', onSelectApp }: ResourcePi
           resources={(allLogicApps ?? []).map((app) => ({
             id: app.id,
             name: app.name,
-            displayName: app.name,
+            displayName: equals(app.plan, 'consumption') ? `${app.name} (Consumption)` : `${app.name} (Standard)`,
```

Comment:
Do we need to localize these strings?

Reply:
I thought these are non localized strings because these are skus... I can update in next PR if that is not the case